File: test03.py
#!/usr/local/bin/python3.5 -u

fib = []
i = 0

while i < 5:
    fib.append(i)
    i += 1

print(', '.join(map(str, fib)))

fib.pop()
fib.pop()

print(', '.join(map(str, fib)))

File: test03.pl
#!/usr/bin/perl -w

@fib = ();
$i = 0;

while ($i < 5) {
    push @fib, $i;
    $i++;
}

print join(', ', @fib), "\n";

pop @fib;
pop @fib;

print join(', ', @fib), "\n";